docs/perf_analyzer.md

```diff
@@ -400,3 +400,29 @@ can achieve by using shared memory in your application. Use
 By default perf_analyzer uses HTTP to communicate with Triton. The GRPC
 protocol can be specificed with the -i option. If GRPC is selected the
 --streaming option can also be specified for GRPC streaming.
+
+## Benchmarking Triton directly via C API
+
```

Comment:
Add 1-2 lines to introduce this feature and how it is useful.  This should be done just before the prerequisites.